docker-compose.yaml

```diff
@@ -0,0 +1,29 @@
+services:
+    cinema:
+        build:
+            context: .
+        ports:
+            - "8000:8000"
+        volumes:
+            -   ./:/app
```

Comment:
Mounting the entire current directory to the `/app` directory in the container can lead to potential issues, such as overwriting files in the container or including unnecessary files. Consider specifying only the necessary directories or files to be mounted.